packages/apps/reputation-oracle/server/src/modules/auth/auth.service.ts

```diff
@@ -187,8 +188,14 @@ export class AuthService {
       reputation_network: this.web3Service.getOperatorAddress(),
     };
 
-    if (userEntity.siteKey) {
-      payload.site_key = userEntity.siteKey.siteKey;
+    // TODO: Check type
+    if (userEntity.siteKeys && userEntity.siteKeys.length > 0) {
+      const hcaptchaSiteKey = userEntity.siteKeys.find(
+        (key) => key.type === SiteKeyType.HCAPTCHA,
+      );
+      if (hcaptchaSiteKey) {
+        payload.site_key = hcaptchaSiteKey.siteKey;
+      }
     }
 
```

Comment:
Can you help me please with above condition? @Dzeranov 
Since `siteKey` has `ManyToOne` relation to user, how we can determine correct key for `auth`?